xaitk_saliency/interfaces/vis_sal_classifier.py

```diff
@@ -65,8 +65,8 @@ def generate(
             Perturbation masks over the reference image.
             This should be parallel in association to the classification
             results input into the `perturbed_conf` parameter.
-            This should have a shape `[nMasks x H x W]`, be float-typed and
```

Comment:
This should not be just integers (see RISE). We maybe want to "relax" the typing requirement similar to the perturb interface, in that we don't specifically require bool, int or float, but indicate that it *may* be either of those three types and the implementation should handle accordingly.

Reply:
Modified docstrings indicate that masks can be either one of the three acceptable formats, and implementations are responsible for handling them.